docs/install.md

```diff
@@ -10,10 +10,12 @@ hide:
 === "pip"
 
     ```sh
-    pip install ibis-framework # (1)
+    pip install 'ibis-framework[duckdb]' # (1) (2)
     ```
 
-    1. Note that the `ibis-framework` package is *not* the same as the `ibis` package in PyPI.  These two libraries cannot coexist in the same Python environment, as they are both imported with the `ibis` module name.
+    1. We suggest starting with the DuckDB backend.  It's performant and fully featured.  If you would like to use a different backend, all of the available options are listed below.
+
+    2. Note that the `ibis-framework` package is *not* the same as the `ibis` package in PyPI.  These two libraries cannot coexist in the same Python environment, as they are both imported with the `ibis` module name.
 
 {% for mgr in ["conda", "mamba"] %}
 === "{{ mgr }}"
```

Comment:
Thanks for the quick PR!

Is it intentional that the conda-forge `ibis-framework` comes by default with the duckdb engine while the pypi does not? I had not realized this discrepancy existed when opening the issue.

Reply:
Not really, except that conda-forge doesn't have a notion of optional dependencies so we decided to make the default install a bit more convenient.

If we did this for pypi installs then duckdb (and a few other things, like sqlalchemy) would become required dependencies.